src/constants/index.js

```diff
@@ -0,0 +1,8 @@
+export const ADD_QUIZ = '/add-quiz';
+export const ADD_QUESTIONS = '/add-questions';
+export const VIEW_QUESTIONS = '/view-questions';
+export const VIEW_QUIZ = '/quiz/:id';
+export const ADD_QUESTION_TYPE = 'ADD_QUESTION';
+export const ADD_QUIZ_TYPE = 'ADD_QUIZ';
+export const ADD_QUIZ_QUESTIONS_TYPE = 'ADD_QUIZ_QUESTIONS';
```

Comment:
You should modularize even your constants, keeping your action type with common constants is a bad practice. Pls separate them into `./state/actions/actionTypes.js`